src/lsp-server.ts

```diff
@@ -334,8 +334,9 @@ export class LspServer {
 
     private runCtags(rootPath: string) {
         const ctagsPath = this.findCtagsPath();
+        const excludeCommands: string = getGitIgnored(rootPath).map(pattern => `--exclude=${pattern}`).join(' ');
         try {
-            execSync(`${ctagsPath} --fields=-anf+iKnS --languages=${CTAGS_SUPPORT_LANGS.join(',')} -R .`, { cwd: rootPath, stdio: 'pipe' });
+            execSync(`${ctagsPath} --fields=-anf+iKnS --languages=${CTAGS_SUPPORT_LANGS.join(',')} ${excludeCommands} -R`, { cwd: rootPath, stdio: 'pipe' });
```

Comment:
I'm not familiar with the ctags usage, why remove the aftermost `.` in the invoke-command?

Reply:
the default argument for `-R` is current path i.e `.`, the reason why I remove it is that when I added the `--exclude`, an empty tags file was generated, so I decided to remove this.